src/structures/Broadcast.js

```diff
@@ -43,7 +43,7 @@ class Broadcast extends Base {
          * Messages statuses
          * @type {Message[]}
          */
-        this.msgs = data.msgs.map(msg => new Message(this.client, msg));
+        this.msgs = data.msgs?.map(msg => new Message(this.client, msg));
```

Comment:
we cant use the "?" operator, it wont work in some older versions of node (I don't recall if we already dropped those versions)
Change to  something like `typeof data.msgs != "undefined" ? data.msgs.map ... : undefined `;

Reply:
@tuyuribr the lib useus Node 18